README.md

```diff
@@ -9,28 +9,29 @@ To run the program you need to use a sbatch script in the HPC. An example of sba
 ```cat script/scheduler_annotation.sbatch```
 ```
 #!/bin/bash
-#SBATCH --job-name=v2g_annotate
+#SBATCH --job-name=v2dg_annotate
 #SBATCH --output=logfile_sbatch.txt
 #SBATCH --partition=cpuq
 #SBATCH --cpus-per-task=5
 #SBATCH --mem=10G
 #SBATCH --time=00:40:00
 
 source /ssu/gassu/miniconda3/etc/profile.d/conda.sh
-conda activate tiledb
-python v2g_dask_query.py -i variants_query.txt -o variants_anno_out.tsv
+conda activate sparkhpc
+
+python main.py --variants_query variants_query.txt --out variants_anno_out
+
 ```
 
-This needs to be modified adding the preferred input and output files. The parameters used can be either using the options:
+This needs to be modified adding the preferred input and output files and the type of annotation that you want to use. The parameters used can be either using the options:
 
```

Comment:
Add a classical usage section here where we report the output of `--help` so all available options are listed and described.

Reply:
I have added another section for helps. What do you think?